### DISCUSSION TITLE
Can I disable/delete the search count like this below?

### DISCUSSION THREAD

**Discussion by neuromaancer** [LazyVim/LazyVim#2233] [Q&A] [answered]
I am using [nvim-hlslens](https://github.com/kevinhwang91/nvim-hlslens) now, but I find when I do a search, there is 
<img width="55" alt="Screenshot 2023-12-21 at 14 48 20" src="https://github.com/LazyVim/LazyVim/assets/28112871/d95aa9ca-75f8-4b6b-9e54-41674d2523ca">
given by `LazyVim`. I don't know how to disable this because `hlslens` already has this function.

**Comment by dpetka2001** [comment, marked as answer]
```lua
-- ~/.config/nvim/lua/plugins/noice.lua
return {
  {
    "folke/noice.nvim",
    opts = {
      messages = {
        view_search = false,
      },
    },
  },
}
```
should do it I think.
